test/workers/process_notification_event_worker_test.rb

```diff
@@ -52,13 +52,19 @@ def test_skip_notifications_for_suspended_account
 
   def test_skip_notifications_from_suspended_buyer
```

Comment:
All looks great. I was looking at the test again. Can we also have the same test but the provider should be suspended and the buyer should be as normal. To verify that buyers would "inherit" the state of the provider.

Reply:
Yeah, it makes no sense because the provider is the one receiving the notification, and it won't receive it if suspended. Anyway, adding such test is a no-brainer copy&paste, so I did it (https://github.com/3scale/porta/pull/4079/commits/393bce43ca39a6d8b00eb8b75b56019ce2d9cc97).

> I wonder if there is any scenario left where a notification for a buyer of a suspended provider would be sent.

In the case that scenario happens now or in the future, we already have a test for it.